balancer-js/src/modules/zaps/bbausd2-migrations/bbausd1.integration.spec.ts

```diff
@@ -129,11 +127,8 @@ describe('bbausd migration execution', async () => {
       network,
       rpcUrl,
     };
-    const subgraph = new Subgraph(config);
-    const pools = new PoolsProvider(
-      config,
-      new SubgraphPoolRepository(subgraph.client)
-    );
+    const networkConfig = getNetworkConfig(config);
+    const pools = new Pools(networkConfig, new Data(networkConfig, provider));
```

Comment:
Here are the main changes related to this merge. Basically replacing the Pools provider and it all worked as expected.